<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>date_widget</class>
 <widget class="QWidget" name="date_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_hour">
   <property name="geometry">
    <rect>
     <x>-9</x>
     <y>12</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>22:36</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ampm">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>13</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_day">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>12</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>30</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_month">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>12</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SEP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_year">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>25</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>2019</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
